portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +31,49 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function loadComments() 
+{
+  fetch('/data').then(response => response.json()).then((tasks) => 
```

Comment:
can you handle error case? e.g. 
`.catch((error) => {
  console.error('Error:', error);
});`
[full api](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)

Reply:
I'm not sure. When I went through the tutorial on the fetch function. It did not mention error cases